airbyte-oauth/src/main/java/io/airbyte/oauth/google/GoogleOAuthFlow.java

```diff
@@ -53,59 +55,53 @@
 
   private final HttpClient httpClient;
 
-  private static final String GOOGLE_ANALYTICS_CONSENT_URL = "https://accounts.google.com/o/oauth2/v2/auth";
-  private static final String GOOGLE_ANALYTICS_ACCESS_TOKEN_URL = "https://oauth2.googleapis.com/token";
-  @VisibleForTesting
-  static final String GOOGLE_ANALYTICS_SCOPE = "https%3A//www.googleapis.com/auth/analytics.readonly";
-  private static final List<String> GOOGLE_QUERY_PARAMETERS = List.of(
-      String.format("scope=%s", GOOGLE_ANALYTICS_SCOPE),
-      "access_type=offline",
-      "include_granted_scopes=true",
-      "response_type=code",
-      "prompt=consent");
+  private final String consentUrl = "https://accounts.google.com/o/oauth2/v2/auth";
+  private final String accessTokenUrl = "https://oauth2.googleapis.com/token";
```

Comment:
I'm curious to know why these were changed from constant strings to attributes to the class?

Is it because you expect in the future to override these values when extending the base class maybe? In that case, is it more clear to initialize these in the constructor (as it is done with `googleQueryParameters`?

Reply:
I initially had them that way but then realized it's probably not going to be injected separately by each connector. Moved it back to static